### PR TITLE
Non standard erc20 delimitation Liquidatable.sol tests

### PR DESCRIPTION
**Pull request by chrismaree**
This is PR 2 of 2, following from [#1159](https://github.com/UMAprotocol/protocol/pull/1159) This includes tests for Liquidatable.sol to show support for non-standard erc20 token types.

### CODE REVIEW

File: core/test/financial-templates/Liquidatable.js [hunk 1]
```diff
@@ -1215,4 +1216,335 @@ contract("Liquidatable", function(accounts) {
...
+          )
+        );
+      });
+      it("Rando calls", async () => {
+        assert(
+          await didContractThrow(
+            USDCLiquidationContract.withdrawLiquidation(liquidationParams.liquidationId, sponsor, { from: rando })
+          )
+        );
+      });
```
Comment (mrice32) [inline core/test/financial-templates/Liquidatable.js:1350]: nit: are these tests relevant to the decimalization?

File: core/test/financial-templates/Liquidatable.js [hunk 2]
```diff
@@ -1215,4 +1216,335 @@ contract("Liquidatable", function(accounts) {
...
+      it("Withdraw still succeeds even if Liquidation has expired", async () => {
```
Comment (mrice32) [inline core/test/financial-templates/Liquidatable.js:1351]: nit: similar to above, is this test relevant?

File: core/test/financial-templates/Liquidatable.js (hunk 1 above)
Comment (chrismaree) [reply core/test/financial-templates/Liquidatable.js:1350]: it's not directly relevant but I wanted to execute a full lifecycle test to ensure that all core logic still operated exactly as expected. If I was to focus on the decimalization only then I'd remove any test that does not involve an assertion on a numerical calculation.

File: core/test/financial-templates/Liquidatable.js (hunk 2 above)
Comment (chrismaree) [reply core/test/financial-templates/Liquidatable.js]: ok I removed all tests that do not directly test raw collateral changing which would be affected by decimalization

Comment (mrice32) [reply core/test/financial-templates/Liquidatable.js]: Sorry, should've probably worded my comments better. I just wanted to better understand how you picked the tests to include vs not include -- wasn't implying that you should necessarily take them out.
